server/middleware/must-be-authenticated.js

```diff
@@ -9,7 +9,7 @@ module.exports = function mustBeAuthenticated(req, res, next) {
     return next();
   }
   if (req.headers.authorization) {
-    return passport.authenticate('basic', { session: false })(req, res, next);
+    return passport.authenticate('basic', { failureRedirect: '/', failureFlash: true })(req, res, next);
```

Comment:
Did this require changing? It has been a while but if I recall this was in place for HTTP basic auth to allow API access using username and password. I'm not sure the failureRedirect/failureFlash do anything here

Reply:
I'll clean this up.